aries-site/src/pages/components/dateinput.mdx

```diff
@@ -0,0 +1,75 @@
+import { Example } from '../../layouts';
+import {
+  DateInputRangeExample,
+  DateInputExample,
+  DateInputSimpleExample,
+  DateInputValidationExample,
+} from '../../examples/components/dateinput';
+
+
+<Example
+  docs="https://v2.grommet.io/dateinput?theme=hpe"
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/dateinput/DateInputSimpleExample.js"
+  figma="https://www.figma.com/file/Mf9if6zyGETTWW5b9PsYNQ/HPE-Date-Input-Component?node-id=1%3A567"
+>
+  <DateInputSimpleExample />
+</Example>
+
+## Guidance
+
+- DateInputs can be used to display past, present, or future dates or times.
+- Use dateinput when you are asking the user for a time or date, or for scheduling tasks.
+
+## When using Date Input, you should:
+
+* Use a label when appropriate
```

Comment:
Accessibility question -- when/how would be the time to use DateInput without a label? Asking because I don't know the answer, not because I don't think it's a viable option.

Reply:
good question maybe we should change that sentence lol